000-temp-trino-aws-proxy/src/main/java/org/eclipse/jetty/client/transport/HttpSender.java

```diff
@@ -0,0 +1,556 @@
+//
```

Comment:
This file seems to be from https://github.com/jetty/jetty.project/pull/11891 with only formatting changes; should we add a comment about this?

Reply:
There is one change and I added a comment. See here: https://github.com/trinodb/aws-proxy/pull/115/files#diff-e49a593f01de37b2d08fffd09c2c9fe1755f45206d789e654408a04ccdfed44aR217